lib/rubocop/cop/style/guard_clause.rb

```diff
@@ -27,6 +27,31 @@ module Style
       #   end
       #
       #   # bad
+      #   def test
+      #     if something
+      #       return foo
+      #     else
+      #       bar
+      #     end
+      #   end
+      #
+      #   # good
+      #   def test
+      #     return foo if something
+      #
+      #     bar
+      #   end
+      #
+      #   # also good
+      #   def test
+      #     if something
+      #       foo
+      #     else
+      #       bar
+      #     end
+      #   end
```

Comment:
I'm worried about the incompatibility to the bad case. What about adding this allowed condition to the cop description text instead of adding an example?